jest.config.js

```diff
@@ -44,6 +44,6 @@ module.exports = {
   verbose: false,
   testPathIgnorePatterns: ['<rootDir>/packages/visx-demo'],
   transformIgnorePatterns: [
-    'node_modules/(?!(d3-(array|color|format|interpolate|scale|time|time-format)|delaunator|internmap|robust-predicates)/)',
+    'node_modules/(?!(d3-(array|color|format|geo|interpolate|scale|time|time-format)|delaunator|internmap|robust-predicates)/)',
```

Comment:
thanks for finding this – sorry I missed it in my instructions! this isn't ideal to have to do for each ESM-only package, will think on ways to streamline it.